x-pack/plugins/observability_solution/investigate_app/public/items/embeddable_item/register_embeddable_item.tsx

```diff
@@ -184,7 +178,18 @@ export function registerEmbeddableItem({
         timeRange: option.globalParams.timeRange,
       };
 
-      return <EmbeddableWidget {...parameters} />;
+      return (
+        <EuiFlexItem
+          grow={true}
+          className={css`
+            > div {
+              height: 128px;
```

Comment:
I had to add this because of a css issue I faced, where the embeddable widget wouldn't show up. 

Similar approach is used in the [`esql` widget](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/investigate_app/public/items/esql_item/register_esql_item.tsx#L216) as well. We should revisit once we use a grid system.

https://github.com/user-attachments/assets/0053cb33-a2b8-4ef2-93ab-febdd9ef390c

Reply:
@kdelemme Here's the [issue](https://github.com/elastic/kibana/issues/192455) for adding `height` to the ItemSchema. Feel free to update the description, I added something basic.